docs/getting-started-with-micro/index.md

```diff
@@ -0,0 +1,286 @@
+
+With Micro, you can see what your data would look like in this format — useful if you want to test any logic that is parsing this data. 
+
+<CodeBlock language="bash">{
+`docker run -p 9090:9090 snowplow/snowplow-micro:${versions.snowplowMicro} --output-tsv`
+}</CodeBlock>
+
+<details>
+<summary>Output vs logs</summary>
+
+The TSV data will be printed to the [standard output](https://en.wikipedia.org/wiki/Standard_streams#Standard_output_(stdout)). As you saw above, Micro also prints logs, which go into the [standard error stream](https://en.wikipedia.org/wiki/Standard_streams#Standard_error_(stderr)).
+
+Depending on how you are running Micro, you might find the logs distracting. If so, you can turn off event logs with an extra option:
+
+<CodeBlock language="bash">{
+`docker run -p 9090:9090 snowplow/snowplow-micro:${versions.snowplowMicro} --output-tsv \\
+  -Dorg.slf4j.simpleLogger.log.EventLog=off`
+}</CodeBlock>
+
+Or just discard the standard error stream entirely using the syntax appropriate for your shell:
+
+<CodeBlock language="bash">{
+`# for bash
+docker run -p 9090:9090 snowplow/snowplow-micro:${versions.snowplowMicro} --output-tsv \\
+  2>/dev/null`
+}</CodeBlock>
+
+</details>
+
+## Advanced usage
+
+### Adding custom schemas
+
+There are two ways to add your own [schemas](/docs/understanding-tracking-design/understanding-schemas-and-validation/index.md) to Micro:
+* By pointing it to your own Iglu resolver
+* By loading your schema files directly into Micro
+
+<Tabs group="adding-schemas">
+  <TabItem value="resolver" label="Using a custom resolver" default>
+
+First, create your [Iglu resolver configuration](/docs/pipeline-components-and-applications/iglu/iglu-resolver/index.md), including your custom resolver. Place it in `iglu.json`.
+
+<details>
+<summary>Example</summary>
+
+```json reference title="iglu.json"
+https://github.com/snowplow-incubator/snowplow-micro/blob/master/example/iglu.json
+```
+
+</details>
+
+Next, you will need to pass this file to the container and instruct Micro to use it.
+
+<CodeBlock language="bash">{
+`docker run -p 9090:9090 \\
+  --mount type=bind,source=$(pwd)/iglu.json,destination=/config/iglu.json \\
+  snowplow/snowplow-micro:${versions.snowplowMicro} \\
+  --iglu /config/iglu.json`
+}</CodeBlock>
+
+  </TabItem>
+  <TabItem value="adding-directly" label="Adding schemas directly">
+
+:::note
+
+This option is only available since version 1.2.0.
+
+:::
+
+Structure your schema file or files like so (this folder layout is significant):
+
+```
+schemas
+└── com.example
+    └── my-schema
+        └── jsonschema
+            ├── 1-0-0
+            └── 1-0-1
```

Comment:
I know it seems obvious, but I got caught up by saving these files as `1-0-0.json` which fails, not sure if worth calling out or not?

Reply:
Yeah, worth calling this out imo.

Reply:
Good point, I actually also fell for that

Reply:
Maybe it's actually something we should support given we all made the mistake 😅